components/AlbumList/Albumlist.js

```diff
@@ -49,14 +57,16 @@ const AlbumList = () => {
   return (
     <AlbumListContainer>
       {albumCovers.map((album) => (
-        <Image
-          alt={album.id}
-          src={album.src}
-          key={album.id}
-          width={140}
-          height={140}
-          style="cover"
-        />
+        <Link href={`/album/${album.id}`} key={album.id}>
+          <Image
+            alt={album.id}
+            src={album.src}
+            key={album.id}
+            width={140}
+            height={140}
+            style="cover"
```

Comment:
Either use the `Image`´s `fill` property or the style property `object-fit: "cover"`.